src/transaction/log_replication.cpp

```diff
@@ -114,7 +116,12 @@ namespace cublog
 	m_minimum_log_lsa.reset (new cublog::minimum_log_lsa_monitor ());
 	// no need to reset with start redo lsa
 
-	m_parallel_replication_redo.reset (new cublog::redo_parallel (replication_parallel, m_minimum_log_lsa.get ()));
+	const bool force_each_log_page_fetch { true };
```

Comment:
It is odd to see a primitive type value initialized with initializer list.
```suggestion
	const bool force_each_log_page_fetch = true;
```